scout/server/blueprints/variants/controllers.py

```diff
@@ -53,7 +53,16 @@ def populate_chrom_choices(form, case_obj):
     form.chrom.choices = [(chrom, chrom) for chrom in chromosomes]
 
 
-def variants(store, institute_obj, case_obj, variants_query, variant_count, page=1, per_page=50):
+def variants(
+    store,
+    institute_obj,
+    case_obj,
+    variants_query,
+    variant_count,
+    page=1,
+    per_page=50,
+    query_form=None,
```

Comment:
Passing only the compound_filter value (an int) or None?

Reply:
Yes, as it is now that would make sense. I am thinking we may sooner or later want to filter on e.g. CADD or mirror the whole variant filter query onto the compounds in which case it makes more sense to pass it all. I could make it a two stage thing I guess, where the query form is first broken down into like a condensed query object and then passed on to the controller.

Reply:
Actually it seems like more fun to try for a more complete filter. I've added a sketch, but the compounds need to be a bit more populated to allow filtering. I'll look into it any day now! 😁